README.md

```diff
@@ -1,21 +1,24 @@
-# SilverStripe Webp Images
+# silverstripe-webp-images
 This SilverStripe module generates webp images from resized jpeg and png images.
 
 ## Requirements
-* SilverStripe ^4.0
-* GD with webp support
+* See `composer.json` requirements
```

Comment:
I think the `GD with webp support` line needs to stay, because you can't set this in `composer.json` as requirement.